app/views/admin/events/_invitation_management.html.haml

```diff
@@ -1,32 +1,34 @@
 - if @event.invitations.any?
   .row
-    .large-12.columns
-      = simple_form_for :invitation, url: admin_invitation_path, method: :put do |f|
-        .medium-6.columns
+    .col
+      .card.border-info
+        .card-body
+          %p.mb-0
+            <strong>#{@event.invitations.count}</strong> members have been invited to this event.
+            %br
+            <strong>#{@attending_students.count}</strong> are attending as students and <strong>#{@attending_coaches.count}</strong> as coaches.
+
+
+  %div.mt-4
```

Comment:
@KimberleyCook not blocking but in HAML you do not need to specify the %div tag. If you do a class or id (`.some-class` or `#some-id`) it will automatically create a div (basically div is the default tag in HAML).

Reply:
Eugh, I do this often and always forget. Damn you HAML!